packages/core/src/lib/__tests__/resource-pill.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { describe, it, expect, vi } from 'vitest';
+import { render, screen, fireEvent } from '@testing-library/svelte';
+import { ResourcePill } from '$lib';
+import { cxTestArguments, cxTestResults } from './cx-test';
+
+describe('ResourcePill', () => {
+  it('Renders text within the resource pill if a value attribute is specified', () => {
+    render(ResourcePill, { value: 'test', icon: 'viam-service' });
+    expect(screen.getByText('test')).toBeVisible();
+  });
+
+  it('Renders a normal resource pill that is removable if removable attribute has been specified', () => {
+    render(ResourcePill, { icon: 'viam-service', removable: true });
+    expect(screen.getByRole('button')).toBeInTheDocument();
```

Comment:
Does this button have a label for accessibility that we can select instead?

Reply:
I will check!